README.md

```diff
@@ -48,3 +53,14 @@ Note: We use a `tollvm12.py` script for multiple reasons:
 Note: Due to snitch's dependency on a custom LLVM-12 backend (which does not support LLVM opaque pointers) we are stuck with MLIR version 16.
 Opaque pointers were introduced in LLVM 15, and support for typed pointers is removed in LLVM 17. 
 More information is available [here](https://llvm.org/docs/OpaquePointers.html). 
+## Run torch-mlir tests
+
+The folder torch include some examples of translating torch models to mlir using the stardew framework.
+The python3.11 installation comes with all the requirements pre-installed and can be run:
+
+```sh
+cd /torch/mult
+python3 main.py
+```
```

Comment:
Please update after this run of the docker container

Reply:
see changes in new commit